compiler/modules/front/front.bal

```diff
@@ -163,25 +163,30 @@ function loadSourcePart(SourcePart part, int i) returns LoadedSourcePart|io:Erro
 }
 
 function imports(s:ModulePart part) returns map<Import>|err:Unimplemented {
-    s:ImportDecl? decl = part.importDecl;
+    s:ImportDecl[]? decl = part.importDecl;
     if decl == () {
         return {};
     }
-    else if decl.org == "ballerina" && decl.module == "io" {
-        string prefix = decl.module;
-        return {
-            [prefix]: {
-                decl,
-                moduleId: { organization: decl.org, names: [decl.module] },
-                defns: ioLibFunctions
-            }
-        };
-    }
     else {
-        return err:unimplemented(`unsupported module ${decl.org + "/" + decl.module}`, loc=err:location(part.file, decl.pos));
+        map<Import> importMap = {};
+        foreach var im in decl {
+            ModuleExports defn;
+            if im.org == "ballerina" && im.module == "io" {
+                defn = ioLibFunctions;
+            }
+            else {
+                // TODO: fix this
+                return err:unimplemented(`unsupported module ${im.module}`, loc=err:location(part.file, im.pos));
+            }
+            importMap[im.module] = {decl:im,
+                moduleId: { organization: im.org, names: im.names.cloneReadOnly()},
+                defns: defn};
+        }
+        return importMap;
     }
 }
 
+
```

Comment:
minor: extra new line

Reply:
fixed

Reply:
apparently not?

Reply:
Sorry I thought this was the extra new line at the end of the file. Both fixed now

Reply:
Extra new line at the end of the file is expected. Better not to remove it.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
What I meant was there were 2 newlines at the end of the file